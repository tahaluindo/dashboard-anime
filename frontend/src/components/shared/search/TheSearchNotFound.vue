<template>
  <p
    class="font-opensans text-center clamptext-08-12 text-primary italic mt-10"
  >
    NÃ£o encontrados resultados para a pesquisa. Tente novamente com outro valor.
  </p>
</template>

<script lang="ts">
import { defineComponent } from "vue";

export default defineComponent({});
</script>

<style lang="postcss"></style>
