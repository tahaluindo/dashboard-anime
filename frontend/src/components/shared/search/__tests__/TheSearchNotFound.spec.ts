import { mount } from "@vue/test-utils";
import TheSearchNotFound from "../TheSearchNotFound.vue";

describe("TheSearchNotFound component test.", () => {
  it("should render the default component.", () => {
    const wrapper = mount(TheSearchNotFound);

    expect(wrapper.html()).toMatchSnapshot();
  });
});
