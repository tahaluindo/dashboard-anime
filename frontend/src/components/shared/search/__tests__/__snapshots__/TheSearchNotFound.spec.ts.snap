// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheSearchNotFound component test. should render the default component. 1`] = `"<p class=\\"font-opensans text-center clamptext-08-12 text-primary italic mt-10\\"> NÃ£o encontrados resultados para a pesquisa. Tente novamente com outro valor. </p>"`;
